packages/code-du-travail-frontend/src/outils/indemniteLicenciement/Wizard.js

```diff
@@ -1,102 +0,0 @@
-import React, { useState } from "react";
```

Comment:
I think it's not a good idea to remove the wizzard since we will have several tools that will be based on it. Otherwise we will duplicate most of the wizzard logic on each tool

Reply:
Ok I didn't know we planned to reuse it 😄